proxied.d.ts

```diff
@@ -1167,9 +1174,12 @@ export interface InputFileProxy<F> {
       tgs_sticker?: F;
       /** WEBM video with the sticker, uploaded using multipart/form-data. See https://core.telegram.org/stickers#video-sticker-requirements for technical requirements */
       webm_sticker?: F;
+      /** Type of stickers in the set. Custom emoji sticker sets can't be created via the Bot API at the moment. By default, a regular sticker set is created */
+      sticker_type?: "regular" | "mask";
       /** One or more emoji corresponding to the sticker */
       emojis: string;
-      /** Pass True, if a set of mask stickers should be created */
+      /** Pass True, if a set of mask stickers should be created
+       * @deprecated Use `sticker_type` instead */
```

Comment:
Same as the other comment, maybe this needs to be removed instead of deprecated.